posthog/api/action.py

```diff
@@ -300,7 +300,7 @@ def _execute_custom_sql(self, query, params):
 
     def _stickiness(self, filtered_events: QuerySet, filters: Dict[Any, Any], request: request.Request) -> Dict[str, Any]:
         date_from, date_to = self._get_dates_from_request(request)
-        range_days = (date_to - date_from).days + 2
+        range_days = (date_to - date_from).days + 2 if date_from else 90
```

Comment:
nit: eventually we should accommodate displaying all time more fully or letting stickiness be filtered by "events that have been repeated between [some time range]".  it doesn't quite matter now because it hasn't even been used for 90 days

Reply:
#625